### PR TITLE
Refactored GOB-Import to use GOBModel dictionary methods.

### PR DESCRIPTION
**Pull request by peteradrichem**

### CODE REVIEW

**Review by roel-devries — DISMISSED** [review]

File: src/gobimport/converter.py [hunk 1]
```diff
@@ -7,20 +7,24 @@
...
 class Converter:
 
     def __init__(self, catalog_name, entity_name, input_spec):
-        self.gob_model = GOBModel()
-        collection = self.gob_model.get_collection(catalog_name, entity_name)
+        self.gob_model = gob_model
+        try:
+            collection = self.gob_model[catalog_name]['collections'][entity_name]
+        except KeyError:
+            collection = None
```
Comment (JKoetsier) [inline src/gobimport/converter.py:27]: I don't like how this pattern is repeated over and over again. It's hard to read, compared to the old simple, very readable method call.

**Review by JKoetsier — CHANGES_REQUESTED** [review]

File: src/gobimport/converter.py (hunk 1 above)
Comment (peteradrichem) [reply src/gobimport/converter.py]: The KeyError try/except can be removed after fixing the tests.

File: src/gobimport/converter.py [hunk 2]
```diff
@@ -71,7 +75,10 @@ def __init__(self, catalogue_name: str, entity_name: str, entity_id_attr: str):
         :param entity_name:
         :param entity_id_attr: The name of the attribute that serves as the entity_id
         """
-        self.collection = GOBModel().get_collection(catalogue_name, entity_name)
+        try:
+            self.collection = gob_model[catalogue_name]['collections'][entity_name]
+        except KeyError:
+            self.collection = None
```
[the remark this answers is not in the document]
Comment (peteradrichem) [reply src/gobimport/converter.py]: Same

File: src/gobimport/reader.py [hunk 3]
```diff
@@ -35,8 +36,10 @@ def __init__(self, source, app, dataset, mode: ImportMode = ImportMode.FULL):
 
         catalogue = dataset['catalogue']
         entity = dataset['entity']
-        gob_attributes = GOBModel().get_collection(catalogue, entity)["all_fields"]
-
+        try:
+            gob_attributes = gob_model[catalogue]['collections'][entity]
+        except KeyError:
+            gob_attributes = None
```
[the remark this answers is not in the document]
Comment (peteradrichem) [reply src/gobimport/reader.py]: Same

File: src/gobimport/validator/__init__.py [hunk 4]
```diff
@@ -281,7 +281,10 @@ def __init__(self, source_app, catalogue, entity_name, input_spec):
         self.source_app = source_app
         self.catalogue = catalogue
         self.entity_name = entity_name
-        self.entity_id = GOBModel().get_collection(self.catalogue, self.entity_name).get('entity_id')
+        try:
+            self.entity_id = gob_model[self.catalogue]['collections'][self.entity_name].get('entity_id')
+        except KeyError:
+            self.entity_id = None
```
[the remark this answers is not in the document]
Comment (peteradrichem) [reply src/gobimport/validator/__init__.py]: Same

**Review by JKoetsier — APPROVED** [review]

**Pull request merged**